keyboard/test/keyboard_spec.rb

```diff
@@ -82,7 +82,7 @@
         allow(Yast::Mode).to receive(:update).and_return false
       end
 
-      it "saves settings to current system" do
+      xit "saves settings to current system" do
```

Comment:
Having a look to the code, maybe this mocking is enough:

~~~
allow(File).to receive(:executable?).with("/usr/sbin/xkbctrl").and_return(false)
~~~

See https://github.com/yast/yast-country/blob/master/keyboard/src/lib/y2keyboard/strategies/kb_strategy.rb#L111